src/Compilers/CSharp/Test/Syntax/Diagnostics/DiagnosticTest.cs

```diff
@@ -1867,11 +1868,197 @@ public static void Main()
                 // (13,25): warning CS1072: Expected identifier or numeric literal.
                 // #pragma warning restore "CS0168
                 Diagnostic(ErrorCode.WRN_IdentifierOrNumericLiteralExpected, @"""CS0168").WithLocation(13, 25),
-                // (14,17): warning CS1634: Expected disable or restore
+                // (14,17): warning CS1634: Expected disable, restore, enable or safeonly
                 // #pragma warning blah
                 Diagnostic(ErrorCode.WRN_IllegalPPWarning, "blah").WithLocation(14, 17));
         }
 
+        [Fact]
+        public void PragmaWarning_EnableSafeOnlyNullable()
+        {
+            var text = @"
+#pragma warning disable nullable
+#pragma warning enable nullable
+#pragma warning restore nullable
+#pragma warning safeonly nullable
+
+#pragma warning disable nullable, 1695
+#pragma warning enable nullable, 1695
+#pragma warning restore nullable, 1695
+#pragma warning safeonly nullable, 1695
+
+#pragma warning enable 
+#pragma warning safeonly
+#pragma warning enable 1695
+#pragma warning safeonly 1695
+";
+            CreateCompilation(text, parseOptions: TestOptions.Regular7_3).VerifyDiagnostics(
+                // (3,17): warning CS1658: Feature 'warning action enable or safeonly' is not available in C# 7.3. Please use language version 8.0 or greater.. See also error CS8370.
```

Comment:
>.. [](start = 157, length = 2)

Extra `.`

Reply:
> Extra .

Preexisting condition. This happens due to the way CS1658 is created. We have several instances in src\Compilers\CSharp\Test\Syntax\Parsing\ParserErrorMessageTests.cs already:
```
// (2,17): warning CS1658: Feature 'generics' is not available in C# 1. Please use language version 2 or greater.. See also error CS8022.
// (2,16): warning CS1658: Feature 'namespace alias qualifier' is not available in C# 1. Please use language version 2 or greater.. See also error CS8022.
// (3,16): warning CS1658: Feature 'namespace alias qualifier' is not available in C# 1. Please use language version 2 or greater.. See also error CS8022.
```


---
In reply to: [245846819](https://github.com/dotnet/roslyn/pull/32208#discussion_r245846819) [](ancestors = 245846819)